src/GenAPI/Microsoft.DotNet.GenAPI.Shared/GenAPIApp.cs

```diff
@@ -0,0 +1,178 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiSymbolExtensions;
+
+namespace Microsoft.DotNet.GenAPI.Shared
+{
+
+    /// <summary>
+    /// Class to standertize initilization and running of GenAPI tool.
+    ///     Shared between CLI and MSBuild tasks frontends.
+    /// </summary>
+    public static class GenAPIApp
+    {
+        public class Context
+        {
+            /// <summary>
+            /// Delimited (',' or ';') set of paths for assemblies or directories to get all assemblies.
+            /// </summary>
+            public string? Assembly { get; set; }
```

Comment:
Shouldn't be nullable but an array instead.

```suggestion
            public string[] Assembly { get; set; }
```